### ISSUE TITLE
Better align Edit VM dialog with VM detail representation

### ISSUE THREAD

**Issue by matobet** [oVirt/ovirt-web-ui#196]
When reviewing the VM detail the user is presented with the following nice detail screen:

![screenshot from 2017-05-22 14-07-18](https://cloud.githubusercontent.com/assets/1173428/26309225/7993d148-3efc-11e7-914c-da7d57f6b783.png)

But when pressing the **edit** action, the whole screen changes to the following form

![screenshot from 2017-05-22 14-11-48](https://cloud.githubusercontent.com/assets/1173428/26309222/778331be-3efc-11e7-82d4-36f4fa7d6d22.png)

I want to propose a more closer aligning of these two, where e.g. the right column could be just swapped with *editable* components and the left part could remain more or less unchanged. This should be quite straightforwardly implementable using the (nested routes) features of react-router.

What do you guys think? @jelkosz @michalskrivanek @mareklibra @vojtechszocs

**Comment by mareklibra** [comment]
Right, this should be aligned to provide sort of "switch" effect by clicking the Edit button.
The VM Detail looks better from these two, so let's come from it.

It's good to note, these screenshots come from https://github.com/oVirt/ovirt-web-ui/issues/156 which is still WIP, _so far no code pushed for review_.

The "VM Detail" needs to be enriched for Template and Cluster fields. 
All field descriptions shall be enriched for the help messages.

The https://github.com/oVirt/ovirt-web-ui/issues/156 changes the UI concept.
To avoid so much rework, I think we should still go with the "Edit VM Component" instead of extending the read-only "VM Detail" component for edit-functionality.
In other words, to maintain the list of fields more or less on two places. This might be considered as duplication of code but really simple one and it offers control over singularities in both flows. The complexity is not in the list itself (just a few lines of dummy code) but in the further processing.

**Comment by mareklibra** [comment]
Follow-up for https://github.com/oVirt/ovirt-web-ui/pull/197
See https://github.com/oVirt/ovirt-web-ui/pull/197#pullrequestreview-40267230

Required for: https://github.com/oVirt/ovirt-web-ui/issues/156

----
Edit VM page needs to be rearranged to:

  - Title: [icon] [VM name] - [Edit]
  - order and design of fields similar to VM Detail

**Comment by mareklibra** [comment]
Blocked by https://github.com/oVirt/ovirt-web-ui/pull/197
